src/utils/common.js

```diff
@@ -555,7 +555,7 @@ export const getFormattedTransactionFeeValue = (feeInWei: string | number | BigN
 };
 
 export const formatTransactionFee = (feeInWei: string | number | BigNumber, gasToken: ?GasToken): string => {
-  if (!feeInWei) return '';
+  if (!feeInWei) return '0';
```

Comment:
IMO it makes more sense to keep the existing code that returned empty string than to return '0'.

Reply:
Thank you for reviewing. I will try to have other work around for NaN issue. I changed this thinking empty string is returning NaN ( not a number) issue. I also looked into one of the screen where NaN was coming and it resolved after changing this condition.